### ISSUE TITLE
not a issue but a question

### ISSUE THREAD

**Issue by cupiditys** [Shaveyy/TwitchSpam#66]
i have a ton of valid `oauth:oauth-token` but the oauth token are invalid how would i generate new ones (everything ive tried resulted in expiring after a few minutes)

**Comment by Shaveyy** [comment]
Question is vague, but if the tokens are expiring it may be that you're logging out of the accounts.

**Comment by cupiditys** [comment]
so how would i get around that? using incognito or clearing cookies instead of logging out still doesn't work

**Comment by Shaveyy** [comment]
Weird, that's normally what I used to do. Are you specifically trying the OAuth's with the bot?

**Comment by cupiditys** [comment]
Yes, it works fine until i login to another account and ihen it stops working
